### PR TITLE
Make sure create does nothing when translation exists & added docs

### PR DESCRIPTION
**Pull request by Nyholm**
Im not too familiar with PhraseApp API. Please correct me if Im wrong.

### CODE REVIEW

File: src/PhraseApp.php [hunk 1]
```diff
@@ -87,12 +93,12 @@ public function create(Message $message)
                 $index = $this->client->translation()->indexKey($this->projectId, $key->getId(), ['tags' => $message->getDomain()]);
                 foreach ($index as $translation) {
                     if ($translation->getLocale()->getId() === $localeId) {
-                        $this->client->translation()->update($this->projectId, $translation->getId(), $message->getTranslation());
```
Comment (Nyholm) [inline src/PhraseApp.php]: `Create` should not update anything if translation exists.

File: src/PhraseApp.php [hunk 2]
```diff
@@ -142,8 +151,14 @@ public function update(Message $message)
                 }
             }
         }
+
+        // No translation was found, lets create one.
+        $this->create($message);
```
Comment (Nyholm) [inline src/PhraseApp.php]: We've forgotten to create, right?

**Comment by prolic** [comment]
I had in mind that create also updates and update should not create. But if was te other way around, sorry, my bad. :-)

**Comment by Nyholm** [comment]
Thank you for merging. 

Great job on this library.

**Comment by prolic** [comment]
You're welcome! I will test again everything tomorrow and if good tag new
releases.

Am 02.02.2017 05:48 schrieb "Tobias Nyholm" <notifications@github.com>:

**Comment by Nyholm** [comment]
Awesome

**Pull request merged**